crates/turbopack-tests/tests/execution/turbopack/resolving/alias-field/input/node_modules/package/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "browser": {
+    "./dir/file1.js": "./replaced/file1.js",
+    "./dir/file2": "./replaced/file2",
+    "dir/file3.js": "./replaced/file3.js",
```

Comment:
I'm not sure if this is correct. I'll take a look at the node.js doc

Reply:
It's not the the spec, and I don't think it's "correct", but webpack supported that and that helps with compatibility

Reply:
I see. Thank you for the explanation!